features/attached_status.feature

```diff
@@ -18,3 +18,123 @@ Feature: Attached status
            | xenial  |
            | impish  |
            | jammy   |
+
+    @series.xenial
+    @series.bionic
+    @series.focal
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attached status in a ubuntu machine
```

Comment:
I really like the idea of consolidating our status checking output into tests like these! I might expand on this further in a future PR.